sopel/irc/backends.py

```diff
@@ -18,7 +18,7 @@
 from sopel.tools.jobs import JobScheduler, Job
 from .abstract_backends import AbstractIRCBackend
 from .utils import get_cnames
-
+from threading import current_thread
```

Comment:
This should be in the "stdlib" import block above. Also, please remember to leave blank lines around the import groups. Example:

```py
import sys
from threading import current_thread

from sopel.tools.jobs import [...]
```

Reply:
agree